scripts/styles.sh

```diff
@@ -6,14 +6,25 @@ OUTPUT_STYLECSS=internal/view/assets/css/style.css
 INPUT_ARCHIVECSS=internal/view/assets/less/archive.less
 OUTPUT_ARCHIVECSS=internal/view/assets/css/archive.css
 
+# Detect support of avx2
+
+if [ -x "$(command -v bun)" ]; then
+    if grep -q avx2 /proc/cpuinfo; then
+        BUN="bun"
+    else
+        BUN="sde -chip-check-disable -- bun"
+        echo "Your cpu not support avx2 so We use sde for more information please lookat https://github.com/oven-sh/bun/issues/762#issuecomment-1186505847"
```

Comment:
Some minor spelling fixes:
```suggestion
        echo "Your CPU does not support avx2 so we use sde, for more information please look at https://github.com/oven-sh/bun/issues/762#issuecomment-1186505847"
```